server/config/index.ts

```diff
@@ -37,6 +37,8 @@ if (process.env.NODE_ENV === 'development') {
 type Config = typeof baseConfig
   & Partial<typeof devConfig>
   & Partial<typeof prodConfig>
-  & Partial<typeof testConfig>
+  & Partial<typeof testConfig>;
 
-export default config;
+const configExport = config;
+
+export default configExport;
```

Comment:
Do these lines do anything different than what was there? It seems like the `configExport` variable is redundant.